shiny/_main.py

```diff
@@ -331,6 +331,13 @@ def run_app(
 
     log_config: dict[str, Any] = copy.deepcopy(uvicorn.config.LOGGING_CONFIG)
 
+    # Workaround for workbench issue 7368
+    deflate_args: DeflateArgs = {}
+    if os.getenv("RS_SERVER_URL"):
```

Comment:
Please import `is_workbench` from https://github.com/posit-dev/py-shiny/blob/2b4d1a0155567b06e300286b6cce33f752f232b2/shiny/_hostenv.py#L13

```suggestion
    if is_workbench():
```